CHANGELOG.md

```diff
@@ -28,15 +28,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   deprecated `TarballUtils`
 -   updated Indriya to version 2.1.2 to include fixes for serialization
 -   Reworking the time series source (one source per time series, distinct mapping source, factory pattern)
+-   BREAKING: Moved methods `buildSafe{Coord,Point,LineString,LineStringBetweenCoords,LineStringBetweenPoints}`, `totalLengthOfLineString` from `GridAndGeoUtils` to `GeoUtils`
```

Comment:
Is this one still valid? Or are these methods now also part of the PSU?

Reply:
Yes, this one is valid. They have been moved to `GeoUtils` in PSU.